tests/bdd/features/ha/target_switchover/target_switchover.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Target Switchover test
+
+  Background:
+    Given a control plane, two Io-Engine instances, two pools
+
+  Scenario: on target switchover older target should not write to replicas
+    Given a volume with two replicas, the volume published on a node
+    When the volume is republished on the other node
+    Then the newer target should serve the io
+    And the older target should not serve the io
+
+  Scenario: if the node hosting the nexus is offline, switchover should not fail
+    Given a volume with two replicas, the volume published on a node
+    When the node hosting the nexus is killed
+    Then the volume republish should not fail
+    Then the newer target should serve the io
+    And the older target should not serve the io
+
+  Scenario: after target switchover if older target is accessible, it should not write to replicas
+    Given a volume with two replicas, the volume published on a node
+    When the node hosting the nexus is killed
+    Then the volume republish should not fail
+    When the node hosting the older nexus is brought back
+    Then the newer target should serve the io
+    And the older target should not serve the io
```

Comment:
If it's killed the older target will no longer exist.
I think here we want to test for temporary network failure, where we can't reach the node to send the shutdown signal, but then network is back and the older nexus is still there?

Reply:
That a great idea, will simulate netwrork failure here. Thanks